layers/+readers/djvu/funcs.el

```diff
@@ -0,0 +1,71 @@
+(defun djvu-toggle-semi-continuous-scrolling ()
+  (interactive)
+  (setq djvu-semi-continuous-scrolling (if djvu-semi-continuous-scrolling
+                                       nil
+                                     t))
+  (message "Djvu alternative scrolling %s" (if djvu-semi-continuous-scrolling
+                                               "enabled"
+                                             "disabled")))
+
+(defun spacemacs/djvu-advise-image-toggle (orig-func &rest args)
```

Comment:
```suggestion
(defun spacemacs/djvu-advise-image-toggle (_file &rest _args)
```

The first argument is `file`.
Since they are not used, they should be prefixed with underscore.

See https://github.com/dalanicolai/djvu2.el/blob/c89f513e589e06bcf239db0f502ed9bac34753d1/djvu.el#L1732